automation/run-tests.sh

```diff
@@ -175,12 +175,12 @@ function rebuild_fed_container_image {
 }
 
 function rebuild_container_images {
-    (is_file_changed "$PROJECT_PATH/packaging" || \
-     is_file_changed "$PROJECT_PATH/automation") \
+    ((is_file_changed "$PROJECT_PATH/packaging" || \
+      is_file_changed "$PROJECT_PATH/automation") \
          &&
-    (rebuild_el7_base_container_image \
-     rebuild_el7_container_image \
-     rebuild_fed_container_image)
+     (rebuild_el7_base_container_image \
+      rebuild_el7_container_image \
+      rebuild_fed_container_image)) || true
```

Comment:
If I understand correctly, the problem is that if the first condition fails, it exists with an error although it should not.
The solution here hides a real problem: If the second condition fails, it should indeed exit with a failure.

I think you should use a simple `if`.

Reply:
OK.